<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ReferenceRangeDock</class>
 <widget class="QWidget" name="ReferenceRangeDock">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>491</width>
    <height>1083</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tabGeneral">
      <attribute name="title">
       <string>General</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="6" column="0">
        <widget class="QLabel" name="lPositionEnd">
         <property name="toolTip">
          <string>Horizontal position relative to label's parent</string>
         </property>
         <property name="text">
          <string>End x:</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="lPosition">
         <property name="font">
          <font>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Position</string>
         </property>
        </widget>
       </item>
       <item row="27" column="0">
        <widget class="QCheckBox" name="chkVisible">
         <property name="text">
          <string>Visible</string>
         </property>
        </widget>
       </item>
       <item row="19" column="2">
        <widget class="QSpinBox" name="sbBackgroundOpacity">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="toolTip">
          <string>The opacity ranges from 0 to 100, where 0 is fully transparent and 100 is fully opaque.</string>
         </property>
         <property name="suffix">
          <string> %</string>
         </property>
         <property name="prefix">
          <string/>
         </property>
         <property name="minimum">
          <number>0</number>
         </property>
         <property name="maximum">
          <number>100</number>
         </property>
         <property name="singleStep">
          <number>10</number>
         </property>
         <property name="value">
          <number>100</number>
         </property>
        </widget>
       </item>
       <item row="23" column="2">
        <widget class="KColorButton" name="kcbBorderColor">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
        </widget>
       </item>
       <item row="18" column="2">
        <widget class="KColorButton" name="kcbBackgroundSecondColor"/>
       </item>
       <item row="12" column="0">
        <widget class="QLabel" name="lBackgroundType">
         <property name="text">
          <string>Type:</string>
         </property>
        </widget>
       </item>
       <item row="17" column="0">
        <widget class="QLabel" name="lBackgroundFirstColor">
         <property name="text">
          <string>First color:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="lName">
         <property name="text">
          <string>Name:</string>
         </property>
        </widget>
       </item>
       <item row="18" column="0" colspan="2">
        <widget class="QLabel" name="lBackgroundSecondColor">
         <property name="text">
          <string>Second color:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Fixed</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>13</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="24" column="0">
        <widget class="QLabel" name="lBorderWidth">
         <property name="text">
          <string>Width:</string>
         </property>
        </widget>
       </item>
       <item row="14" column="0">
        <widget class="QLabel" name="lBackgroundColorStyle">
         <property name="text">
          <string>Style:</string>
         </property>
        </widget>
       </item>
       <item row="12" column="2">
        <widget class="QComboBox" name="cbBackgroundType">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
        </widget>
       </item>
       <item row="0" column="2">
        <widget class="QLineEdit" name="leName">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="lComment">
         <property name="text">
          <string>Comment:</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item row="26" column="0">
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>694</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="5" column="0">
        <widget class="QLabel" name="lPositionStart">
         <property name="toolTip">
          <string>Horizontal position relative to label's parent</string>
         </property>
         <property name="text">
          <string>Start x:</string>
         </property>
        </widget>
       </item>
       <item row="23" column="0">
        <widget class="QLabel" name="lBorderColor">
         <property name="text">
          <string>Color:</string>
         </property>
        </widget>
       </item>
       <item row="19" column="0">
        <widget class="QLabel" name="lBackgroundOpacity">
         <property name="text">
          <string>Opacity:</string>
         </property>
        </widget>
       </item>
       <item row="16" column="0">
        <widget class="QLabel" name="lBackgroundBrushStyle">
         <property name="text">
          <string>Style:</string>
         </property>
        </widget>
       </item>
       <item row="6" column="2">
        <widget class="QLineEdit" name="lePositionEnd">
         <property name="showClearButton" stdset="0">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="13" column="2">
        <widget class="QLineEdit" name="leBackgroundFileName">
         <property name="toolTip">
          <string>Specify the name of the image file.</string>
         </property>
         <property name="clearButtonEnabled">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="17" column="2">
        <widget class="KColorButton" name="kcbBackgroundFirstColor"/>
       </item>
       <item row="16" column="2">
        <widget class="QComboBox" name="cbBackgroundBrushStyle"/>
       </item>
       <item row="15" column="0">
        <widget class="QLabel" name="lBackgroundImageStyle">
         <property name="text">
          <string>Style:</string>
         </property>
        </widget>
       </item>
       <item row="8" column="2">
        <widget class="QDateTimeEdit" name="dtePositionEnd"/>
       </item>
       <item row="13" column="3">
        <widget class="QPushButton" name="bOpen">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="toolTip">
          <string> Select the file to import</string>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="1" column="2">
        <widget class="ResizableTextEdit" name="teComment">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
        </widget>
       </item>
       <item row="4" column="2">
        <widget class="KComboBox" name="cbOrientation"/>
       </item>
       <item row="14" column="2">
        <widget class="QComboBox" name="cbBackgroundColorStyle"/>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="lOrientation">
         <property name="text">
          <string>Orientation:</string>
         </property>
        </widget>
       </item>
       <item row="13" column="0">
        <widget class="QLabel" name="lBackgroundFileName">
         <property name="text">
          <string>File name:</string>
         </property>
        </widget>
       </item>
       <item row="21" column="0">
        <widget class="QLabel" name="lBorder">
         <property name="font">
          <font>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Border</string>
         </property>
        </widget>
       </item>
       <item row="11" column="0">
        <widget class="QLabel" name="lBackground">
         <property name="font">
          <font>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Background</string>
         </property>
        </widget>
       </item>
       <item row="7" column="2">
        <widget class="QDateTimeEdit" name="dtePositionStart"/>
       </item>
       <item row="5" column="2">
        <widget class="QLineEdit" name="lePositionStart">
         <property name="showClearButton" stdset="0">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="22" column="0">
        <widget class="QLabel" name="lBorderStyle">
         <property name="text">
          <string>Style:</string>
         </property>
        </widget>
       </item>
       <item row="8" column="0">
        <widget class="QLabel" name="lPositionDateTimeEnd">
         <property name="text">
          <string>End x:</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <spacer name="verticalSpacer_5">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Fixed</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>18</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="24" column="2">
        <widget class="QDoubleSpinBox" name="sbBorderWidth">
         <property name="suffix">
          <string> pt</string>
         </property>
         <property name="singleStep">
          <double>0.500000000000000</double>
         </property>
        </widget>
       </item>
       <item row="25" column="2">
        <widget class="QSpinBox" name="sbBorderOpacity">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="toolTip">
          <string>The opacity ranges from 0 to 100, where 0 is fully transparent and 100 is fully opaque.</string>
         </property>
         <property name="suffix">
          <string> %</string>
         </property>
         <property name="prefix">
          <string/>
         </property>
         <property name="minimum">
          <number>0</number>
         </property>
         <property name="maximum">
          <number>100</number>
         </property>
         <property name="singleStep">
          <number>10</number>
         </property>
         <property name="value">
          <number>100</number>
         </property>
        </widget>
       </item>
       <item row="7" column="0">
        <widget class="QLabel" name="lPositionDateTimeStart">
         <property name="text">
          <string>Start x:</string>
         </property>
        </widget>
       </item>
       <item row="10" column="0">
        <spacer name="verticalSpacer_16">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Fixed</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>18</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="20" column="0">
        <spacer name="verticalSpacer_18">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Fixed</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>72</width>
           <height>18</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="15" column="2">
        <widget class="QComboBox" name="cbBackgroundImageStyle"/>
       </item>
       <item row="22" column="2">
        <widget class="QComboBox" name="cbBorderStyle">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
        </widget>
       </item>
       <item row="25" column="0">
        <widget class="QLabel" name="lBorderOpacity">
         <property name="text">
          <string>Opacity:</string>
         </property>
        </widget>
       </item>
       <item row="28" column="0">
        <spacer name="verticalSpacer_17">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="9" column="2">
        <widget class="QComboBox" name="cbPlotRanges"/>
       </item>
       <item row="9" column="0">
        <widget class="QLabel" name="lPlotRange">
         <property name="text">
          <string>Plot range:</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KComboBox</class>
   <extends>QComboBox</extends>
   <header>kcombobox.h</header>
  </customwidget>
  <customwidget>
   <class>KColorButton</class>
   <extends>QPushButton</extends>
   <header>kcolorbutton.h</header>
  </customwidget>
  <customwidget>
   <class>ResizableTextEdit</class>
   <extends>QTextEdit</extends>
   <header>src/kdefrontend/widgets/ResizableTextEdit.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
